includes/cli-workload-version.md

```diff
@@ -0,0 +1,7 @@
+---
+ms.date: 11/12/2025
+ms.topic: include
+---
+- **`version <WORKLOAD_VERSION>`**
+
+  A workload version to display, or one or more workloads and their versions joined by the `@` character.  Support for `@` character available since 9.0.200 SDK.
```

Comment:
```suggestion
  A workload version to display, or one or more workloads and their versions joined by the `@` character.  Support for the `@` character is available since 9.0.200 SDK.
```